pages/api/setup/[token]/index.ts

```diff
@@ -0,0 +1,58 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import jackson from '@lib/jackson';
+
+export const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+  const { method } = req;
+
+  switch (method) {
+    case 'POST':
+      return handlePOST(req, res);
+    case 'GET':
+      return handleGET(req, res);
+    default:
+      res.setHeader('Allow', ['GET', 'POST']);
+      res.status(405).json({ data: null, error: { message: `Method ${method} Not Allowed` } });
+  }
+};
+
+// Create a new setup link
```

Comment:
There are 2 places I can see ` await setupLinkController.create()` being called. First at `pages/api/admin/setup-links/index.ts` and second at `/pages/api/setup/[token]/index.ts`. I'm not sure if this is really needed. Setup link creation API should be placed within `/api/admin/`. cc @deepakprabhakara Right?